packages/animations/lib/src/open_container.dart

```diff
@@ -423,34 +413,34 @@ class _OpenContainerRoute extends ModalRoute<void> {
       _FlippableTweenSequence<double>(<TweenSequenceItem<double>>[
     TweenSequenceItem<double>(
       tween: Tween<double>(begin: 1.0, end: 0.0),
-      weight: 4 / 12,
+      weight: 1 / 5,
     ),
     TweenSequenceItem<double>(
       tween: ConstantTween<double>(0.0),
-      weight: 8 / 12,
+      weight: 4 / 5,
     ),
   ]);
   static final _FlippableTweenSequence<double> _fadeThroughOpenOpacityTween =
       _FlippableTweenSequence<double>(<TweenSequenceItem<double>>[
     TweenSequenceItem<double>(
       tween: ConstantTween<double>(0.0),
-      weight: 4 / 12,
+      weight: 1 / 5,
     ),
     TweenSequenceItem<double>(
       tween: Tween<double>(begin: 0.0, end: 1.0),
-      weight: 8 / 12,
+      weight: 4 / 5,
     ),
   ]);
 
   final TweenSequence<Color> _scrimFadeInTween = TweenSequence<Color>(
```

Comment:
(not part of your PR, but maybe you can fix it): These two could also be statics.

Reply:
Done